src/drivers/uart_esc/uart_esc.cpp

```diff
@@ -90,9 +90,9 @@ input_rc_s 		_rc;
 pwm_limit_t	_pwm_limit;
 
 // esc parameters
-uint16_t _pwm_disarmed;
```

Comment:
@julianoes The wrong variable size might have been a contributing factor. But not sure.